### ISSUE TITLE
UNIX time is not shown correctly

### ISSUE THREAD

**Issue by fukuchi** [tzinfo/tzinfo#91]
`Timezone#strftime("%s", time)` converts the given time to UNIX time. Because UNIX time is the number of seconds that have elapsed since 1970-01-01 00:00:00 (UTC), the result does not depends on the timezone.

However, the result of the folloing code seems incorrect.

``` ruby
#!/usr/bin/ruby

require 'tzinfo'

jst = TZInfo::Timezone.get("Japan")
utc = TZInfo::Timezone.get("UTC")

time = Time.now.utc

puts jst.strftime("%F %T (%Z): %s", time)
puts utc.strftime("%F %T (%Z): %s", time)
```

The code returned:

```
2018-08-30 21:18:23 (JST): 1535663903
2018-08-30 12:18:23 (UTC): 1535631503
```

In the case above, I expecet the result as:

```
2018-08-30 21:18:23 (JST): 1535631503
2018-08-30 12:18:23 (UTC): 1535631503
```

I used tzinfo-1.2.3 with Ruby 2.3.

**Comment by philr** [comment]
Thanks for the report.

This is already resolved in the upcoming version 2.0.0 release (currently on the master branch).

For earlier versions of TZInfo, this is a consequence of returning UTC instances of Time and DateTime to represent local times in order to support Ruby versions older than 1.9.3 (that can only handle times in the local system time zone and UTC).

I've committed a change to the 1.2 branch to make `Timezone#strftime` return the correct number of seconds since the epoch for the %s directive.

**Comment by fukuchi** [comment]
Thank you for the quick response! Great help.
